java/test/jmri/jmrix/loconet/LnThrottleManagerTest.java

```diff
@@ -20,9 +21,12 @@
 public class LnThrottleManagerTest extends jmri.managers.AbstractThrottleManagerTestBase {
 
     private DccThrottle throttle;
+    private DccThrottle throttle2;
 
     boolean failedThrottleRequest = false;
+    boolean failedThrottleRequest2 = false;
     int flagGotStealRequest = -1;
+    int flagGotStealRequest2 = -1;
```

Comment:
Since this is only used in a single test, I think this should be a local variable in that test

Reply:
Since these variables are assigned values within the listener, they need to be fields, not local variables.

Reply:
Ok, thanks for checking.